spec/support/page_objects/sections/vacancy.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require_relative "base"
+require "forwardable"
+
+module PageObjects
+  module Sections
+    class Vacancy < PageObjects::Sections::Base
```

Comment:
Maybe we should just call this GovCheckbox, what do you think?

Reply:
will do so when we need to re-use it in another page